### PR TITLE
Clean up repo, add documentation and examples

### PR DESCRIPTION
**Pull request by philippe-eecs**
## Summary

- Update pyproject.toml with proper authors and Na-VAE URLs
- Rewrite README.md with usage examples, comparison table, and citation
- Add HTML blog post (docs/index.html) explaining DINO vs LPIPS background
- Add gradient visualization figure showing where each loss focuses
- Add usage_example.py showing autoencoder training integration
- Add fig_gradients.py (Modal) for generating gradient visualization
- Add distortions.py with common image distortion functions
- Remove old/unused files (dinov3.py, compare_dino_lpips.py)
- Clean up examples directory structure

## Key additions

**README.md** - Install instructions, quick start, autoencoder example, comparison table:

| Metric | Pixel-only | + LPIPS | + DINO |
|--------|------------|---------|--------|
| rFID | 2.13 | 0.72 | **0.30** |
| rFDD | 6.96 | 2.93 | **1.12** |

**docs/index.html** - Blog post explaining:
- Background on perceptual losses
- Why DINO over LPIPS
- Empirical results from ViTok ablations
- Gradient visualization

**examples/usage_example.py** - Shows integration in training loop

## Test plan

- [ ] `pip install -e .` works
- [ ] Blog post renders correctly
- [ ] Gradient figure displays properly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,77 +1,127 @@
...
 
 ```python
-from dino_perceptual import DINOv3Model
+import torch
+import torch.nn as nn
+from dino_perceptual import DINOPerceptual
+
+# Initialize models
+autoencoder = MyAutoencoder().cuda()
+perceptual_loss = DINOPerceptual(model_size="B").cuda().eval()
```
Comment (philippe-eecs) [inline README.md]: add bfloat16 and torch.compile fullgraph=True

File: README.md [hunk 2]
```diff
@@ -1,77 +1,127 @@
...
+    total_loss = l1_loss + 250.0 * dino_loss
+
+    # Backward pass
+    optimizer.zero_grad()
+    total_loss.backward()
+    optimizer.step()
 ```
 
-## API Reference
+## Why DINO over LPIPS?
```
Comment (philippe-eecs) [inline README.md]: Add a small paragraph about how Dino is trained with more update to date arch more images etc.

**Comment by philippe-eecs** [comment]
LGTM
